doc/visual-programming/source/widgets/data/mergedata.md

```diff
@@ -12,24 +12,79 @@ Merges two datasets, based on values of selected attributes.
 
 - Data: dataset with features added from extra data
 
-The **Merge Data** widget is used to horizontally merge two datasets, based on values of selected attributes. In the input, two datasets are required, data and extra data. The widget allows selection of an attribute from each domain, which will be used to perform the merging. The widget produces one output. It corresponds to instances from the input data to which attributes from input extra data are appended.
+The **Merge Data** widget is used to horizontally merge two datasets, based on the values of selected attributes (columns). In the input, two datasets are required, data and extra data. The widget allows selection of one or more attributes from each domain, which will be used to perform the merging. The widget produces one output. It corresponds to the instances from the input data to which attributes (columns) from input extra data are appended.
```

Comment:
> The widget allows selection of one or more attributes from each domain, which will be used to perform the merging. 

Maybe: **Rows from the two data sets are matched by the values of pairs of attributes, chosen by the user.**